app/components/Search/SearchPage.js

```diff
@@ -13,7 +13,7 @@ type Props = {
   onQueryChanged: string => void,
   placeholder?: string,
   results: Array<SearchResult>,
-  handleSelect: SearchResult => void
+  handleSelect: SearchResult => Promise<*>
```

Comment:
Hehe, you fixed this in:
https://github.com/webkom/lego-webapp/pull/1270/files
, and will get a merge conflict if either of these pull requests go through.